SYCL/ESIMD/histogram_raw_send.cpp

```diff
@@ -7,7 +7,7 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: gpu-intel-gen9
-// UNSUPPORTED: gpu-intel-dg1,cuda,hip
+// UNSUPPORTED: gpu-intel-dg1,gpu-intel-dg2,cuda,hip
```

Comment:
This whole line looks odd and... redundant. I'd think it can be just removed.
Doesn't the line 9 require gen9 and prohibit from running on dg1,dg2,cuda,hip ?

Reply:
I'd assume so but there is a regression report indicating it's failing on dg2. 
If you are sure line 9 is sufficient to exclude the test on other platform, then we shouldn't need any fix.